inc/term-sidebars.php

```diff
@@ -1,51 +1,65 @@
 <?php
+/**
+ * File for Largo's Term Sidebars
+ *
+ * @package Largo
+ */
 
 /**
  * Display the fields for selecting icons for terms in the "post-type" taxonomy
  */
 class Largo_Term_Sidebars {
 
-	function __construct() {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
 		add_action( 'edit_category_form_fields', array( $this, 'display_fields' ) );
 		add_action( 'edit_tag_form_fields', array( $this, 'display_fields' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts') );
-		add_action( 'edit_terms', array( $this, 'edit_terms' ) );
-		add_action( 'create_term', array( $this, 'edit_terms' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		add_action( 'edit_terms', array( $this, 'edit_terms' ), 10, 2 );
+		add_action( 'create_term', array( $this, 'edit_terms' ), 10, 2 );
 	}
 
 	/**
+	 * Return a list of taxonomies that term sidebars are enabled for
 	 *
+	 * @filter largo_get_sidebar_taxonomies
+	 * @return Array of taxonomy slugs
 	 */
-	function get_sidebar_taxonomies() {
-		if ( empty($this->_sidebar_taxonomies) ) {
+	public function get_sidebar_taxonomies() {
```

Comment:
This shouldn't be public; it has no parameters.

Reply:
I'm against changing this.

Methods with undeclared visibility are public so changing this would be a breaking change if someone is using the currently-public method. And the method itself doesn't need parameters; it returns the output of a filter.